src/en/credentials.md

```diff
@@ -198,14 +203,21 @@ juju credentials --format yaml --show-secrets
 
 The YAML output will be similar to our 'mycreds.yaml' sample above.
 
-You can set the default credential:
+### Setting default credentials
+
+You can set the default credential for a cloud:
```

Comment:
This only works for locally stored credentials. I think we should be here about this distinction here too.

Reply:
:ok: